os/fs/driver/mtd/smart.c

```diff
@@ -5163,7 +5164,8 @@ static int smart_journal_checkout(FAR struct smart_struct_s *dev, journal_log_t
  * Name: smart_journal_process_transaction
  *
  * Description:
- *   Check out journal entry, checkpoint to mtd header from journal data
+ *   Process the COMMIT/ERASE/RELEASE task onto MTD flash memory based on
```

Comment:
MTD is kind of driver, not memory, so instead of MTD flash memory, I would like to use something like a 'through the mtd driver'

Reply:
Modified